worlds/witness/player_logic.py

```diff
@@ -414,7 +414,7 @@ def handle_postgame(self, world: "WitnessWorld") -> List[List[str]]:
         postgame_adjustments = []
 
         # Make some quick references to some options
-        remote_doors = world.options.shuffle_doors >= 2  # "Panels" mode has no region accessibility implications.
+        remote_doors = world.options.shuffle_doors >= "doors"  # "Panels" mode has no region accessibility implications.
```

Comment:
Hardcoding string constants like this should be discouraged imo. Similarly the original code of using magic values should also be discouraged imo.

I think it's much better to use `world.options.shuffle_doors >= world.options.shuffle_doors.option_doors` or to import the option module/class itself and do either `world.options.shuffle_doors >= ShuffleDoors.option_doors` or `world.options.shuffle_doors >= options.ShuffleDoors.option_doors`.

Reply:
Well we already support every single other method of string comparison and multiple documents use them as examples. I don't think leaving out one particular method of comparing is doing any good

Reply:
> I think it's much better to use world.options.shuffle_doors >= world.options.shuffle_doors.option_doors

Disagree, this is pretty cursed.

> world.options.shuffle_doors >= ShuffleDoors.option_doors

Correct, this is best.

Strings are officially supported though, this has been extensively discussed & ultimately made clear by the core maintainer team. 
There are protections in place. These comparisons actually **crash** if the string is e.g. misspelled. 
Also, string comparisons are already supported for `==`, so either we should get rid of that, or add this. This weird inbetween world where `== str` is supported but `< str` isn't is bad, imo.

Reply:
> String comparisons are officially supported though, this has been extensively discussed & ultimately made clear by the core maintainer team. There are protections in place. These comparisons actually **crash** if the string is e.g. misspelled. Also, string comparisons are already supported for `==`, so either we should get rid of that, or add this. This weird inbetween world where `== str` is supported but `< str` isn't, is the worst world, imo.

Respectfully, I disagree with this assessment. A string isn't a numerical value. when you're doing `==` you're doing it with the class, not the integer so it does make sense, but when you're doing `<` or `>` you're now doing arithmetic with strings.